cql/ManageRareAbnormality.cql

```diff
@@ -673,6 +684,17 @@ define MostRecentCin2BiopsyInPast2Years:
     )
   )
 
+define MostRecentHsilBiopsyInPast6Years:
```

Comment:
Can we specifically call this "Unspecified HSIL"? An HSIL Biopsy is a more broad term that could mean CIN2, CIN3, or HSIL, Unspecified. An Example of this type of labelling is at https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ManageRareAbnormality.cql#L642.

Reply:
I see "HsilBiopsy" is now changed to "HsilUnspecifiedBiopsy" throughout this file. Thanks!